rebar.config.script

```diff
@@ -7,10 +7,10 @@ case erlang:function_exported(rebar3, main, 1) of
         %% profiles
         CONFIG_DEPS =
         [{deps, [{meck, ".*",{git, "https://github.com/eproxus/meck.git", {tag, "0.9.0"}}},
-                 {jsx, ".*", {git, "git://github.com/talentdeficit/jsx.git", {tag, "v2.11.0"}}},
+                 {jsx, ".*", {git, "https://github.com/talentdeficit/jsx.git", {tag, "v2.11.0"}}},
                  %% {hackney, ".*", {git, "git://github.com/benoitc/hackney.git", {tag, "1.2.0"}}},
                  {eini, ".*", {git, "https://github.com/erlcloud/eini.git", {tag, "1.2.9"}}},
-                 {lhttpc, ".*", {git, "git://github.com/erlcloud/lhttpc", {tag, "1.6.2"}}},
+                 {lhttpc, ".*", {git, "https://github.com/erlcloud/lhttpc", {tag, "1.6.2"}}},
```

Comment:
I just realized this is the only dep without .git on the end. that seems pointlessly different. Do you mind to fix it?

Reply:
Will do.